pkg/data/management/role_data.go

```diff
@@ -481,6 +490,9 @@ func BootstrapAdmin(management *wrangler.Context) (string, error) {
 		if err != nil && !apierrors.IsAlreadyExists(err) {
 			return "", errors.Wrap(err, "can not ensure admin user exists")
 		}
+		if err == nil {
+			logrus.Infof("Default username/password created, username: admin, password: %s", bootstrapPassword)
```

Comment:
Probably more safe to write the password to a file in the container? If logs are forwarded they end up somewhere. In general I think we avoid logging sensitive data unless its in TRACE.

Reply:
It's not sensitive any more than knowing the current hardcoded "admin" is, you have to change it on setup.

(ok change password doesn't    _actually_ require it to be different, you can change that if you want)

This should be as easy to use as possible, which means a `https://<server-url or hostname>/?setup=<password>` URL (that terminals will make clickable).

Reply:
Yeah, this is changed to https://<server-url or hostname>/dashboard/?setup=<password>, looks like UI respect this setup query and auto-forward to change password page.